tests/integration/eZ/API/RichTextFieldTypeIntegrationTest.php

```diff
@@ -681,7 +679,37 @@ public function testExternalLinkStoringAfterUpdate(): void
             $content->contentInfo->currentVersionNo
         );
 
-        $this->assertNotContains(reset($urlIds), $urlIdsAfterUpdate);
+        $urlId = $this->getUrlIdForLink($testLink);
+
+        $this->assertContains($urlId, $urlIdsAfterUpdate);
+    }
+
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \Doctrine\DBAL\Driver\Exception
+     * @throws \Doctrine\DBAL\Exception
+     * @throws \ErrorException
+     */
+    private function getUrlIdForLink(string $link): int
+    {
+        $connection = $this->getRawDatabaseConnection();
+        $query = $connection->createQueryBuilder();
+        $query
+            ->select(
+                $connection->quoteIdentifier('id')
+            )
+            ->from('ezurl')
```

Comment:
```suggestion
            ->from(\eZ\Publish\Core\Persistence\Legacy\URL\Gateway\DoctrineDatabase::URL_TABLE)
```
(but import FQCN please)